cl/people_db/import_judges/populate_fjc_judges.py

```diff
@@ -641,3 +641,73 @@ def make_federal_judge(item, testing=False):
                 position.save()
             except Exception:
                 continue
+
+
+def make_mag_bk_judge(item, testing=False):
+    """Takes the federal judge data <item> and associates it with a Judge object.
+    Returns a Judge object.  To be used for importing bankruptcy and magistrate
+    judges.
+    """
+
+    if not testing:
```

Comment:
I'd take this whole section out of the `if not testing` block. Since it doesn't save to the DB, I'd let it run whether testing or not.